shared/src/main/scala/mlscript/JSBackend.scala

```diff
@@ -185,6 +185,7 @@ class JSBackend {
     case IntLit(value) => JSLit(value.toString + (if (JSBackend isSafeInteger value) "" else "n"))
     case DecLit(value) => JSLit(value.toString)
     case StrLit(value) => JSExpr(value)
+    case UnitLit(value) => if(value){JSExpr("undefined")} else{JSExpr("null")}
```

Comment:
This actually creates a JS string literal, which is not what we want. I believe what we want is `JSLit(if (value) "undefined" else "null")`.